epub33/core/index.html

```diff
@@ -4092,348 +4092,376 @@ <h4>Prefixed Properties</h4>
-								</tr>
-								<tr>
-									<td>
-										<code>text-transform</code>
-									</td>
-									<td>
-										<code>-epub-fullwidth</code>
-									</td>
-									<td>
-										<code>text-transform: full-width</code>
-									</td>
-								</tr>
-							</tbody>
-						</table>
+						
+						<section id="sec-css-prefixed-text-epub-hyphens">
+						<h6>The <code>-epub-hyphens</code> Property</h6>
+						<p>This is a prefixed version of the <code>hyphens</code> property.</p>
+							<table class="def propdef">
+								<tbody>
+									<tr>
+										<th>Name: </th>
+										<td>
+											<dfn>-epub-hyphens</dfn>
+										</td>
+									</tr>
+									<tr class="value">
+										<th>Value:</th>
+										<td>
+											none | manual | auto | all
+										</td>
+									</tr>
+								</tbody>
+							</table>
+							<p class="note">The <code>auto</code> value is no longer supported in CSS.</p>
+						</section>
+						<section id="sec-css-prefixed-text-epub-line-break">
+						<h6>The <code>-epub-line-break</code> Property</h6>
+						<p>This is a prefixed version of the <code>line-break</code> property.</p>
+							<table class="def propdef">
+								<tbody>
+									<tr>
+										<th>Name: </th>
+										<td>
+											<dfn>-epub-linebreak</dfn>
+										</td>
+									</tr>
+									<tr class="value">
+										<th>Value:</th>
+										<td>
+											auto | loose | normal | strict
+										</td>
+									</tr>
+								</tbody>
+							</table>
+						</section>
+						<section id="sec-css-prefixed-text-epub-text-align-last">
+						<h6>The <code>-epub-text-align-last</code> Property</h6>
+						<p>This is a prefixed version of the <code>text-align-last</code> property</p>
+							<table class="def propdef">
+								<tbody>
+									<tr>
+										<th>Name: </th>
+										<td>
+											<dfn>-epub-text-align-last</dfn>
+										</td>
+									</tr>
+									<tr class="value">
+										<th>Value:</th>
+										<td>
+											auto | start | end | left | right | center | justify
+										</td>
+									</tr>
+								</tbody>
+							</table>
+						</section>
+						<section id="sec-css-prefixed-text-epub-word-break">
+						<h6>The <code>-epub-word-break</code> Property</h6>
+						<p>This is a prefixed version of the <code>word-break</code> property.</p>
+							<table class="def propdef">
+								<tbody>
+									<tr>
+										<th>Name: </th>
+										<td>
+											<dfn>-epub-word-break</dfn>
+										</td>
+									</tr>
+									<tr class="value">
+										<th>Value:</th>
+										<td>
+											normal | keep-all | break-all
+										</td>
+									</tr>
+								</tbody>
+							</table>
+						</section>
+						<section id="sec-css-prefixed-text-text-transform">
+						<h6>The <code>text-transform</code> Property</h6>
+						<p>This is a prefixed value for the <code>text-transform</code> property.</p>
+							<table class="def propdef">
```

Comment:
This is an exception... it is _not_ a prefixed property! I guess it should be treated differently and how exactly it relates to the CSS equivalent...

Reply:
Correct. The section heading describes the text-transform property. The text describes it as a prefixed value of that property. The table shows what the CSS equivalent is. I'm not sure what you're asking me to change.

Reply:
Ouch, sorry, my bad. It indeed says 'prefixed _value_' (emphasis mine).

(If I want to be picky: the section heading says 'Prefixed Properties', and this section refers to a prefixed value... But that is something we can let go, I guess...)